ElementX/Sources/Other/Pills/PillViewModel.swift

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright 2023 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Combine
+import Foundation
+
+enum PillViewState {
+    case loadingUser(userID: String)
+    case loadedUser(userID: String, name: String, avatarURL: URL?)
```

Comment:
Maybe not important right now, but if we wanted to future proof this, might want to be more like
```suggestion
    case loading(PillType)
    case loaded(type: PillType, name: String, avatarURL: URL?)
```

Reply:
Do I even need the pilltype actually? maybe not i could just have contentID in it